.circleci/config.yml

```diff
@@ -316,8 +316,8 @@ jobs:
         default: 9
       cadence:
         type: enum
-        enum: [ "ci", "daily", "weekly" ]
-        default: "ci"
+        enum: ['ci', 'daily', 'weekly']
```

Comment:
Can you revert these formatting changes?

Reply:
Reverted the formatting changes